fastlane_core/lib/fastlane_core/keychain_importer.rb

```diff
@@ -19,17 +19,22 @@ def self.import_file(path, keychain_path, keychain_password: "", certificate_pas
       if Helper.backticks('security -h | grep set-key-partition-list', print: false).length > 0
         command = "security set-key-partition-list"
         command << " -S apple-tool:,apple:"
+        command << " -l 'Imported Private Key'"
```

Comment:
could you explain this additional param?

Reply:
Before the `set-key-partition-list` operation would apply to all keys in the keychain, potentially taking lots of time (see https://github.com/fastlane/fastlane/issues/12291#issuecomment-383595648) if the user has lots of keys.

The `-l` parameter will filter the operation to keys that match the supplied "label". As far as I know, that label "Imported Private Key" is applied by the previous step in the import.